.gitignore

```diff
@@ -0,0 +1,41 @@
+# Temporary and binary files
```

Comment:
I prefer not to have this file. See [ignoring artifacts](https://blog.jaraco.com/skeleton/#ignoring-artifacts).

Reply:
Sorry for that, I haven't read this recommendation before.

I have removed the `.gitignore` file now.